tasks/src/app/simple-button/simple-button.component.html

```diff
@@ -1 +1,3 @@
-<button (click)="increseNumberOfCounts()">Number of clicks {{numberOfClicks}}</button>
\ No newline at end of file
+<button (click)="changeInputStatus()">{{ inputStatus?("Disable"):("Enable") }}</button><input type="text"
```

Comment:
```suggestion
<button (click)="changeInputStatus()">{{ inputStatus ? "Disable" : "Enable" }}</button><input type="text"
```
insert spaces for readability and get rid of parentheses